src/seedu/addressbook/data/person/Email.java

```diff
@@ -37,10 +35,7 @@ public static boolean isValidEmail(String test) {
         return test.matches(EMAIL_VALIDATION_REGEX);
     }
 
-    @Override
-    public String toString() {
-        return value;
-    }
+   
```

Comment:
you can also consider pushing the constructor and the validation logic to the parent class.